upgrade/patches/43_api_token.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE {{%api_token}} (
+  apt_id int unsigned NOT NULL auto_increment,
+  apt_usr_id int unsigned NOT NULL,
+  apt_created datetime NOT NULL,
+  apt_status varchar(10) NOT NULL default 'active',
+  apt_token varchar(32) NOT NULL,
+  PRIMARY KEY (apt_id),
+  KEY (apt_token)
```

Comment:
altho probably for security, should probably never include token in the sql query (except unavoidable INSERT), and if comparing for for value select all active tokens for usr_id, they key should be usr_id+apt_status instead.